design/ceph/object/ceph-rgw-support-multisite-for-external-cluster.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
No need for the separators in the header, those are just for the generated docs, not the design docs
```suggestion
```